addons/website/static/src/scss/website.scss

```diff
@@ -2446,3 +2446,43 @@ input[value*="data-oe-translation-initial-sha"] {
 .o_dropdown_without_offset {
     transform: none !important;
 }
+
+// Subtle input-group-text
+// This can be set to an input-group-text placed directly before or after the input.
+.input-group-text.input-group-text-subtle {
+    @include transition($input-transition);
+    border-color: $input-border-color;
+    background: $input-bg;
+    color: $input-color;
+
+    > * {
+        color: rgba($input-color, 0.6);
+    }
```

Comment:
Should be documented I don't understand the point of that rule

Reply:
This is to make the decorative element more subtle. I should document this line 👍 .
<img width="321" alt="Capture d’écran 2024-02-13 à 16 06 17" src="https://github.com/odoo/odoo/assets/80679690/2fac0fff-d909-4445-a562-06547397c3eb">

Reply:
But why `> *` ? Why not changing the color of the whole element ? (the line just above that `> *` rule)

Reply:
For now, the input border color is based on `currentColor`. This won't work if we apply the color to the whole element.
<img width="312" alt="Capture d’écran 2024-02-14 à 07 56 11" src="https://github.com/odoo/odoo/assets/80679690/33d2f0fb-1024-4798-afcd-ff02f6c1e54f">

We should probably do something about this border color in another PR.

Reply:
Oh, I see, justifiable in a comment but this is indeed a bit annoying: if you want to add text in that subtle element, you have to remember you have to put it in a span or something like that :grimacing:

Reply:
I could add more precision to my comment, what do you think?
https://github.com/odoo/odoo/pull/153374/files#diff-f338780f594d6b1f46eda5d3fb9fed86f3c3c8f65522fb93e639534e9d92cdb2R2458-R2459

Reply:
Yes you should, the important thing to comment is not what it does (that I can understand reading the code) but why `> *` is needed instead of defining a `color` directly on the element itself (which is not straightforward, at least for me reading it) :+1:

Reply:
I updated the comment, what do you think ? 
https://github.com/odoo/odoo/compare/8719ee70b84976409b29050f501007335d90b01b..e07ce0c9d691764d2d179e22f950b67dd3f7e21d

Reply:
A point at the end and I will be super happy :wink: